### ISSUE TITLE
[New Output] Swift Testing Results

### ISSUE THREAD

**Issue by tonyarnold** [cpisciotta/xcbeautify#313]
### What's the uncaptured output?

✔ Test run with 16 tests passed after 0.004 seconds.
◇ Test run started.
↳ Testing Library Version: 94 (arm64-apple-ios13.0-simulator)
◇ Suite UUIDRFC4122Tests started.
◇ Test "Test that a directly initialized UUID outputs in lowercase" started.
◇ Test "Test that a UUID initialized with an uppercase UUID string outputs in lowercase" started.
✔ Test "Test that a directly initialized UUID outputs in lowercase" passed after 0.008 seconds.
✔ Test "Test that a UUID initialized with an uppercase UUID string outputs in lowercase" passed after 0.008 seconds.
✔ Suite UUIDRFC4122Tests passed after 0.011 seconds.
✔ Test run with 30 tests passed after 0.012 seconds.

### Additional Context

I haven't captured the test failure, timeout and other states here in this output.

**Comment by piercifani** [comment]
We've had to remove XCBeautify from our CI pipeline after migrating to Swift Testing because of this, interested in giving a hand if needed

**Comment by clausjoergensen** [comment]
I can confirm we're seeing the same:

without xcbeautify:

```
Testing started
Test Suite 'Selected tests' started at 2024-09-23 11:26:17.705.
Test Suite 'MyUnitTests.xctest' started at 2024-09-23 11:26:17.706.
Test Suite 'MyUnitTests.xctest' passed at 2024-09-23 11:26:17.706.
	 Executed 0 tests, with 0 failures (0 unexpected) in 0.000 (0.000) seconds
Test Suite 'Selected tests' passed at 2024-09-23 11:26:17.706.
	 Executed 0 tests, with 0 failures (0 unexpected) in 0.000 (0.001) seconds
◇ Test run started.
↳ Testing Library Version: 94 (arm64-apple-ios13.0-simulator)
◇ Iteration 1 started.
◇ Suite MyViewModelTests started.
✔ Suite MyViewModelTests passed after 0.001 seconds.
◇ Iteration 1 ended after 0.001 seconds.
✔ Test run with 0 tests passed after 0.001 seconds.
```

vs. with xcbeautify

```
Selected tests
MyUnitTests.xctest
Test Succeeded
```

**Comment by dagostini** [comment]
@cpisciotta I can take this one on, since we need it fixed for our own CI/CD, but I can't commit more than 6h per week to it.

**Comment by cpisciotta** [comment]
Hi everyone! Thank you for your patience, and sorry for the delay! @ajkolean recently opened a PR to address this issue, and I will work with him to get this merged ASAP.

**Comment by tonyarnold** [comment]
No need to apologise, @cpisciotta! This is a great tool, and I'm more than happy to wait when I don't have time to directly contribute. Thank you!

**Comment by pepicrft** [comment]
Thank you folks for addressing this one 🙏🏼

**Comment by cpisciotta** [comment]
Hey everyone! Thank you again to @ajkolean for his contribution! This should be available starting in 2.12.0, but that is now several versions back from the latest. I will close this issue, but please re-open this or a new issue if you continue to see areas of improvement.

**Comment by tonyarnold** [comment]
Thanks @ajkolean and @cpisciotta 🫶
